apps/mymedicare_cb/tests/test_callback.py

```diff
@@ -188,6 +174,43 @@ def catchall(url, request):
             # assert login
             self.assertNotIn('_auth_user_id', self.client.session)
 
+    def test_callback_url_success(self):
+        # mock sls user info endpoint
+        @urlmatch(netloc='dev.accounts.cms.gov', path='/v1/oauth/userinfo')
+        def sls_user_info_mock(url, request):
+            return {
+                'status_code': 200,
+                'content': {
+                    'sub': '0744b8c2-53ff-4288-91e5-96eb14703fda',
+                    'given_name': 'Clarissa',
+                    'family_name': 'Dalloway',
+                    'email': 'bob@bobserver.bob',
+                    'hicn': '1234567890A',
+                },
+            }
+
+        self._callback_url_success(sls_user_info_mock)
+
+        # assert new user's name is Clarissa Dalloway
+        user = User.objects.get(username="0744b8c2-53ff-4288-91e5-96eb14703fda")
```

Comment:
add an assertion that this user exists to the `test_callback_url_no_names_success` test.